<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabserver</class>
 <widget class="QWidget" name="tabserver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0">
     <item row="0" column="0">
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="15" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>接收</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>清空计数</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>服务器名</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>连接类型</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>总计数</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="2">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>TCP服务端</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UDP单播</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UDP广播</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UDP组播</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>指定本机IP</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>指定本机端口</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>启动</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>目标端口</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>目标IP</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="6" column="3">
         <widget class="QPushButton" name="pushButton_14">
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>接收区</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="12">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 170, 255);</string>
          </property>
          <property name="text">
           <string> 01..........05.............10.............15.............20.............25.............30.............35.............40.............45.............50</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="12">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 170, 255);</string>
          </property>
          <property name="text">
           <string> 01..........05.............10.............15.............20.............25.............30.............35.............40.............45.............50</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4" colspan="4">
         <widget class="QLineEdit" name="lineEdit_9"/>
        </item>
        <item row="2" column="4" colspan="5">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>发送文件</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>转发设置</string>
          </property>
         </widget>
        </item>
        <item row="8" column="6" colspan="6">
         <widget class="QPushButton" name="pushButton_13">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>监视模式</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="3">
         <widget class="QCheckBox" name="checkBox_8">
          <property name="text">
           <string>显示查找到的帧</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2" colspan="2">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>自动发送</string>
          </property>
         </widget>
        </item>
        <item row="2" column="11">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>选项</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10" rowspan="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item row="7" column="9">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="12">
         <widget class="QTextEdit" name="textEdit_3">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::WidgetWidth</enum>
          </property>
          <property name="lineWrapColumnOrWidth">
           <number>0</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="3">
         <widget class="QCheckBox" name="checkBox_7">
          <property name="text">
           <string>保存到文件(实时)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>16进制发送</string>
          </property>
         </widget>
        </item>
        <item row="2" column="10">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item row="7" column="10" colspan="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>表格显示</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/icon/table.png</normaloff>:/new/prefix1/icon/table.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="8" colspan="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="12">
         <widget class="QTextEdit" name="textEdit_2">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="12">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>暂停显示</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4" colspan="2">
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>帧过滤设置</string>
          </property>
         </widget>
        </item>
        <item row="0" column="11" rowspan="2">
         <widget class="QPushButton" name="pushButton_7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>发送区</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7" rowspan="2">
         <widget class="QLineEdit" name="lineEdit_8"/>
        </item>
        <item row="0" column="4" rowspan="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>间隔</string>
          </property>
         </widget>
        </item>
        <item row="6" column="6" colspan="2">
         <widget class="QCheckBox" name="checkBox_9">
          <property name="text">
           <string>转发</string>
          </property>
         </widget>
        </item>
        <item row="6" column="9" colspan="3">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>16进制显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
